whisper/docker-entrypoint.sh

```diff
@@ -17,7 +17,7 @@ then
     echo "ERROR: Must specify an environment variable SERVICE_MODE in [ http | task | websocket ] (None was specified)"
     exit -1
 else
-    if [ "$SERVICE_MODE" = "http" ] 
+    if [[ "$SERVICE_MODE" == "http" && "$ENABLE_STREAMING" != "true" ]]
```

Comment:
Are you sure you want to avoid running ingress when the streaming route is enabled ?

Reply:
OK it seems to be "Remove http+ENABLE_STREAMING for Whisper. It will redirect to websocket mode"
Why do you want to do this ? I think I missed sth